libminifi/test/unit/YamlConfigurationTests.cpp

```diff
@@ -457,3 +456,64 @@ Flow Controller:
   REQUIRE(NULL != rootFlowConfig->findProcessor("XYZ")->getUUID());
   REQUIRE(!rootFlowConfig->findProcessor("XYZ")->getUUIDStr().empty());
 }
+
+class DummyComponent : public core::ConfigurableComponent {
```

Comment:
let's leave override out of libminifi/ and keep it in extensions due to compiler versioning.